Modules/Core/Mesh/wrapping/itkImageToMeshFilter.wrap

```diff
@@ -1,4 +1,5 @@
 itk_wrap_include("itkMesh.h")
```

Comment:
in commit message `... it is also to generate a
Mesh.` should perhaps be _... it is also **able** to generate a
Mesh._